clouddriver-huaweicloud/src/main/java/com/netflix/spinnaker/clouddriver/huaweicloud/cache/Keys.java

```diff
@@ -77,59 +78,50 @@ private static String getCloudProviderId() {
     return HuaweiCloudProvider.ID;
   }
 
+  private static Map emptyMap() {
```

Comment:
you had this in your other PR too, just curious what does this buy you (vs. just calling `Collection.emptyMap` which itself is already a `final static` (https://hg.openjdk.java.net/jdk8/jdk8/jdk/file/687fd7c7986d/src/share/classes/java/util/Collections.java#l4557)

I don't care/think it matters either way, but curious why you went this route

Reply:
I just don't like to write Collections.emtyMap so many times. Any better suggestions? thanks!